frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditIconModal/EditIconModal.tsx

```diff
@@ -0,0 +1,369 @@
+      >
+        <Card
+          borderRadiusSize="xxlarge"
+          className={`${baseClass}__preview-card__fleet`}
+        >
+          <div className={softwareDetailsBaseClass}>
+            {iconPreviewUrl ? (
+              <img
+                src={iconPreviewUrl}
+                alt="Uploaded icon preview"
+                style={{ width: 96, height: 96 }}
+              />
+            ) : (
+              <SoftwareIcon name={software.name} source="apps" size="xlarge" />
+            )}
+            <dl className={infoClass}>
+              <h1>{name}</h1>
+              <dl className={descriptionListClass}>
+                {!!type && <DataSet title="Type" value={type} />}
+
+                {!!previewInfo.versions && (
+                  <DataSet title="Versions" value={versions} />
+                )}
+              </dl>
+            </dl>
+          </div>
+          <div className={resultsCountClass}>
+            <TableCount name="versions" count={versions} />
+            {countsUpdatedAt && TitleVersionsLastUpdatedInfo(countsUpdatedAt)}
+          </div>
+          <div className={`data-table-block ${baseClass}__preview-table`}>
+            <div className="data-table data-table__wrapper">
+              <table className="data-table__table">
+                <thead>
+                  <tr role="row">
+                    <th
+                      className="version__header"
+                      colSpan={1}
+                      role="columnheader"
+                    >
+                      <div className="column-header">Version</div>
+                    </th>
+                    <th
+                      className="vulnerabilities__header"
+                      colSpan={1}
+                      role="columnheader"
+                    >
+                      <div className="column-header">Vulnerabilities</div>
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr className="single-row" role="row">
+                    <td className="version__cell" role="cell">
+                      88.0.1
+                    </td>
+                    <td className="vulnerabilities__cell" role="cell">
+                      <div
+                        className="vulnerabilities-cell__vulnerability-text-with-tooltip"
+                        data-tip="true"
+                        data-for="86"
+                      >
+                        <span className="text-cell w250 italic-cell">
+                          620 vulnerabilities
+                        </span>
+                      </div>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </Card>
+        <div
+          className={`${baseClass}__mask-overlay ${baseClass}__mask-overlay--fleet`}
+        />
+      </Card>
+    );
+  };
+
+  const renderPreviewSelfServiceCard = () => {
```

Comment:
Preview Self-service section was more complicated to mock out static html (has a search bar and a side nav menu, etc) so I took a screenshot of the more difficult section, white out the icon and name and then imposed the icon and name over the screenshot.